### ISSUE TITLE
MDCAlertController should announce when it is presented in VoiceOver

### ISSUE THREAD

**Issue by sread94** [material-components/material-components-ios#1622]
When an MDCAlertController is presented, VoiceOver should say the word "Alert".

**Comment by ianegordon** [comment]
Hi Sarah, thanks for reporting the issue. I think the main delay has been supporting internationalization for the VO announcement. I'm going to point this at the original issue, but please feel free to comment there.

Duplicate of #821
